src/pages/latest/delta-update.mdx

```diff
@@ -410,24 +410,31 @@ Here is a detailed description of the `merge` programmatic operation.
-| Target columns: `key`, `value` <br></br> Source columns:` key`, `newValue`          | `targetDeltaTable.alias("t") .merge( sourceDataFrame.alias("s"), "t.key = s.key") .whenMatched().updateAll() .whenNotMatched().insertAll() .execute()`                                                          | `updateAll` and `insertAll` actions throw an error because the target column `oldValue` is not in the source.                                                                                                                                                       | The table schema is changed to `(key, oldValue, newValue)`. `updateAll` updates columns `key` and `newValue` leaving `oldValue` unchanged, and `insertAll` inserts rows `(key, NULL, newValue)` (that is, `oldValue` is inserted as `NULL`).                        |
-| "Target columns: `key`, `value` <br></br> Source columns:` key`, `newValue`"        | `targetDeltaTable.alias("t") .merge( sourceDataFrame.alias("s"), "t.key = s.key") .whenMatched().update(Map( "newValue" -> col("s.newValue"))) .whenNotMatched().insertAll() .execute()`                        | `update` throws an error because column `newValue` does not exist in the target table.                                                                                                                                                                              | `update` still throws an error because column `newValue` does not exist in the target table.                                                                                                                                                                        |
-| Target columns: `key`, `value` <br></br> Source columns:` key`, `newValue`          | `targetDeltaTable.alias("t") .merge( sourceDataFrame.alias("s"), "t.key = s.key") .whenMatched().updateAll() .whenNotMatched().insert(Map( "key" -> col("s.key"), "newValue" -> col("s.newValue"))) .execute()` | `insert` throws an error because column `newValue` does not exist in the target table.                                                                                                                                                                              | `insert` still throws an error as column `newValue` does not exist in the target table.                                                                                                                                                                             |
```

Comment:
```
Target columns: `key`, `value`
```
This is confusing, this should be:
```
Target columns: key, oldValue
```
which the actual doc does show properly, but I don't see where the right text would come from:

https://docs.delta.io/latest/delta-update.html